lib/app_navigator.dart

```diff
@@ -57,6 +66,11 @@ class AppNavigatorImpl extends AppNavigator {
     context.go('$_routePathRootScreen$_routePathHomeScreen');
   }
 
+  @override
+  void navigateToLoginScreen({required BuildContext context}) {
+    context.go('$_routePathRootScreen$_routePathLoginScreen');
```

Comment:
@Shayokh144 In Android, the user can press the "psychical back button" to navigate back to the previous page.

Since we don't want to navigate back to `AppStarterScreen`, we should use `replace` instead of `go`:

```suggestion
    context.replace('$_routePathRootScreen$_routePathLoginScreen');
```

📔 The same goes for `navigateToHomeScreen(...).

Otherwise, this will happen:

https://user-images.githubusercontent.com/18277915/233930561-d21d8e29-d679-48e9-a6dc-11836b4d86d8.mov

Reply:
Agree 👍